region/tourian/main/Metroid Room 4.json

```diff
@@ -127,6 +127,7 @@
         ]},
         "h_canCrystalFlash"
       ],
+      "resetsObstacles": ["A"],
```

Comment:
Not liking this one. I'm thinking the resetRoom is instead supposed to be a come in normally option

Reply:
Ok, split that off into a separate strat.